src/gateway/kong-enterprise/developer-portal/publish-spec.md

```diff
@@ -0,0 +1,28 @@
+---
+title: How to Publish an API Spec to the Dev Portal
+badge: enterprise
+content_type: how-to
+---
+
+Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.
+
+## Publish a Spec to Dev Portal
```

Comment:
```suggestion
```
I think you might not need this heading since it would just be repeating the title, but I'm not sure what the precedent is here about omitting h2 before a task